main.js

```diff
@@ -0,0 +1,68 @@
+localStorage.clear()
+
+const apiManager = new APIManager()
+const renderer = new Renderer()
+
+
+
+$('#loadButton').on('click',function(){
+    apiManager.getUsers()
+    apiManager.getKanyeQuote()
+    apiManager.getRandomPoke()
+    apiManager.getMeatTalk()
+})
+
+
+const diaplsyData = function(dataObj){
+
+    const mainUser = dataObj.mainUsers
+    renderer.renderMainUser(mainUser)
+
+    const friendsUsersArray = dataObj.friendUsers 
+    renderer.renderFriendUsers(friendsUsersArray)
+
+    const kanyeQuote = dataObj.kanyeQuote
+    renderer.renderKnyeQuote(kanyeQuote)
+
+    const pokemon = dataObj.poke
+    renderer.renderPoke(pokemon)
+
+    const meatTalk = dataObj.meatTalk
+    renderer.getMeatTalk(meatTalk)
+}
+
+
+
+$('#displayButton').on('click',function(){
+    diaplsyData(apiManager.data)
+})
+
+
+
+$('#saveUserButton').on('click', function(){
+    const currentUserData = apiManager.data
+    const usersName = (currentUserData.mainUsers.name.title + ' ' + currentUserData.mainUsers.name.first + ' ' + currentUserData.mainUsers.name.last)
+    let savedAlredy = false
+    const savedUsers = $('option')
+    for(user of savedUsers){
```

Comment:
When declaring a new variable you should declare it with _const_ or _let_, otherwise, the variable will be defined as "_var_"